### PR TITLE
Add Karaf Cellar releases 4.0.5 and 4.1.2

### PR DESCRIPTION
**Pull request by fpapon**
@jbonofre I pushed a single PR for the both releases and may be we can update the Cellar series in the download pages by adding Cellar 4.2.x futur series and mark Cellar 4.0.x series as maintenance.

Thoughts ?

### CODE REVIEW

File: src/main/webapp/archives.html [hunk 1]
```diff
@@ -558,12 +558,20 @@ <h3 class="text-dark">Karaf Cellar</h3>
...
-                        </td>
-                        <td><a href="javascript:var w = window.open('https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12311140&version=12340615')">Release Notes</a></td>
+                            <td>4.1.1</td>
+                            <td><a href="javascript:var w = window.open('download.html')">Installation Instructions</a></td>
+                            <td><a href="http://archive.apache.org/dist/karaf/cellar/4.1.1/apache-karaf-cellar-4.1.1-src.tar.gz">tar.gz</a> [<a href="http://archive.apache.org/dist/karaf/cellar/4.1.1/apache-karaf-cellar-4.1.1-src.tar.gz.asc">PGP</a>] [<a href="http://archive.apache.org/dist/karaf/cellar/4.1.1/apache-karaf-cellar-4.1.1-src.tar.gz.md5">MD5</a>] [<a href="http://archive.apache.org/dist/karaf/cellar/4.1.1/apache-karaf-cellar-4.1.1-src.tar.gz.sha1">SHA1</a>]<br>
```
Comment (jbonofre) [inline src/main/webapp/archives.html:563]: That's not correct anymore. Per "new" Apache rule, `md5` is not used anymore and `sha1` is replaced by `sha512`.

**Review by jbonofre — CHANGES_REQUESTED** [review]
We now have to use `sha512` instead of `md5` and `sha1`. It's what I'm now using on dist.apache.org.

File: src/main/webapp/archives.html (hunk 1 above)
Comment (fpapon) [reply src/main/webapp/archives.html:563]: Ok, I made the changes on the Cellar 4.1.1 archives.

**Comment by fpapon** [comment]
Ok, I made the change on the archives of Cellar 4.1.1.

File: src/main/webapp/archives.html [hunk 2]
```diff
@@ -558,12 +558,20 @@ <h3 class="text-dark">Karaf Cellar</h3>
...
-                        </td>
-                        <td><a href="javascript:var w = window.open('https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12311140&version=12340615')">Release Notes</a></td>
+                            <td>4.1.1</td>
+                            <td><a href="javascript:var w = window.open('download.html')">Installation Instructions</a></td>
+                            <td><a href="http://archive.apache.org/dist/karaf/cellar/4.1.1/apache-karaf-cellar-4.1.1-src.tar.gz">tar.gz</a> [<a href="http://archive.apache.org/dist/karaf/cellar/4.1.1/apache-karaf-cellar-4.1.1-src.tar.gz.asc">PGP</a>] [<a href="http://archive.apache.org/dist/karaf/cellar/4.1.1/apache-karaf-cellar-4.1.1-src.tar.gz.sha512">SHA512</a>]<br>
+                                        <a href="http://archive.apache.org/dist/karaf/cellar/4.1.1/apache-karaf-cellar-4.1.1-src.zip">zip</a> [<a href="http://archive.apache.org/dist/karaf/cellar/4.1.1/apache-karaf-cellar-4.1.1-src.zip.asc">PGP</a>] [<a href="http://archive.apache.org/dist/karaf/cellar/4.1.1/apache-karaf-cellar-4.1.1-src.zip.sha512">SHA512</a>]
```
Comment (jbonofre) [inline src/main/webapp/archives.html]: Sorry my bad, in archive, we should still use sha1/md5 (4.1.1 is the last one using this).

File: src/main/webapp/download.html [hunk 3]
```diff
@@ -87,16 +87,16 @@ <h3 class="mb-0 text-dark">Karaf Runtime <span class="text-muted">4.2.1</span></
...
               </p>
               <p class="card-text mb-auto">
                 Source Distribution :
-                  <a href="http://www.apache.org/dyn/closer.lua/karaf/cellar/4.1.1/apache-karaf-cellar-4.1.1-src.tar.gz">tar.gz</a> [<a href="https://www.apache.org/dist/karaf/cellar/4.1.1/apache-karaf-cellar-4.1.1-src.tar.gz.asc">PGP</a>] [<a href="https://www.apache.org/dist/karaf/cellar/4.1.1/apache-karaf-cellar-4.1.1-src.tar.gz.sha1">SHA1</a>] - 
-                                <a href="http://www.apache.org/dyn/closer.lua/karaf/cellar/4.1.1/apache-karaf-cellar-4.1.1-src.zip">zip</a> [<a href="https://www.apache.org/dist/karaf/cellar/4.1.1/apache-karaf-cellar-4.1.1-src.zip.asc">PGP</a>] [<a href="https://www.apache.org/dist/karaf/cellar/4.1.1/apache-karaf-cellar-4.1.1-src.zip.sha1">SHA1</a>]
+                  <a href="http://www.apache.org/dyn/closer.lua/karaf/cellar/4.1.2/apache-karaf-cellar-4.1.2-src.tar.gz">tar.gz</a> [<a href="https://www.apache.org/dist/karaf/cellar/4.1.2/apache-karaf-cellar-4.1.2-src.tar.gz.asc">PGP</a>] [<a href="https://www.apache.org/dist/karaf/cellar/4.1.2/apache-karaf-cellar-4.1.2-src.tar.gz.sha512">SHA512</a>] -
```
Comment (jbonofre) [inline src/main/webapp/download.html:98]: This is correct.

**Review by jbonofre — CHANGES_REQUESTED** [review]

**Comment by fpapon** [comment]
NP ;) 
It's good I change for MD5 on the 4.1.0 of Cellar.

**Review by jbonofre — APPROVED** [review]

**Pull request merged**